### PR TITLE
scc 2.7.0 (new formula)

### PR DESCRIPTION
**Pull request by elindsey**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Adding a new formula for scc. Looks like someone attempted to submit this last year, but a binary only build - this is a proper source build.

### CODE REVIEW

File: Formula/scc.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/boyter/scc/").install Dir["*"]
+    system "go", "build", "-o", "#{bin}/scc", "-v", "github.com/boyter/scc/"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/scc --version")
```
Comment (javian) [inline Formula/scc.rb]: Can you modify the test so it tests the functionality of the software ?

Comment (elindsey) [reply Formula/scc.rb]: Sure thing! I've adapted the test from a similar tool (cloc.rb).

**Review by javian — APPROVED** [review]

**Comment by elindsey** [comment]
Ping for merge? 🙂

**Comment by elindsey** [comment]
Ping for merge @javian @lembacon ? 🙂

**Comment by elindsey** [comment]
All approved - could this be merged?

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Oh sorry, one more thing: can you please squash everything into a single commit, with commit message `scc 2.7.0 (new formula)`? Then we can merge

**Comment by elindsey** [comment]
All squashed 🙂

**Comment by fxcoudert** [comment]
Thank you for your contribution to Homebrew @elindsey!

**Pull request closed without merging**